pkg/tracker/tracker.go

```diff
@@ -28,15 +28,16 @@ import (
 )
 
 const HTTP_TIMEOUT = 5 * time.Second
-const GA_API_URL = "https://www.google-analytics.com/collect"
 const GA_TRACKING_ID = "GA_TRACKING_ID"
+const GA_API_URL = "GA_API_URL"
```

Comment:
Can we please reduce this block ?

```
const (
     HTTP_TIMEOUT = 5 * time.Second
    GA_TRACKING_ID string = "GA_TRACKING_ID"
     GA_API_URL string = "GA_API_URL"
)
```
Also please add types for constants as appropriate.